src/components/KYCWall/BaseKYCWall.js

```diff
@@ -15,90 +15,112 @@ import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import {defaultProps, propTypes} from './kycWallPropTypes';
 
+const POPOVER_MENU_ANCHOR_POSITION_HORIZONTAL_OFFSET = 20;
```

Comment:
I would suggest to add a comment to describe why the const exists.

Reply:
This was already there and was requested to be moved to a CONST in Previous discussion. Can't tell exactly why it's added. I'm only fixing a refactor issue. https://github.com/Expensify/App/pull/28798#discussion_r1345650106